Mathlib/Data/List/Enum.lean

```diff
@@ -34,23 +34,22 @@ theorem get_enum (l : List α) (i : Fin l.enum.length) :
     l.enum.get i = (i.1, l.get (i.cast enum_length)) := by
   simp
 
-theorem mk_add_mem_enumFrom_iff_get? {n i : ℕ} {x : α} {l : List α} :
+@[deprecated mk_add_mem_enumFrom_iff_getElem? (since := "2024-08-12")]
+theorem mk_add_mem_enumFrom_iff_get? {n i : Nat} {x : α} {l : List α} :
```

Comment:
I don't really care about this, but I'd just like to know the answer from you, since there was a PR that did the opposite at one point recently but you didn't seem to mind. Why are switching from `ℕ` to `Nat` here? You can just `@` me with the answer and resolve this.

Reply:
@j-loreaux. Oh, purely a copy-paste lapse on my part. I'd had this chunk running in the Lean4 folder, where I couldn't use `ℕ`, and didn't notice when I moved it back here.

I'll fix now.